meta-lmp-base/recipes-support/htpdate/htpdate_1.3.6.bb

```diff
@@ -28,4 +32,8 @@ do_compile () {
 
 do_install () {
 	oe_runmake install 'INSTALL=install' 'STRIP=echo' 'DESTDIR=${D}'
+	if ${@bb.utils.contains('DISTRO_FEATURES', 'systemd', 'true', 'false', d)}; then
+		install -d ${D}${systemd_unitdir}/system
+		install -m 644 ${S}/scripts/htpdate.service ${D}${systemd_unitdir}/system
```

Comment:
`${D}${systemd_system_unitdir}` is the same as `${D}${systemd_unitdir}/system` in a single variable.

Reply:
Done, I always forget about this one.